packages/botonic-react/src/webchat-app.jsx

```diff
@@ -163,12 +168,18 @@ export class WebchatApp {
   }
 
   addBotMessage(message) {
+    message.ack = 0
+    message.isUnread = true
+    message.sentBy = message.sent_by?.split('message_sent_by_')[1]
+    delete message.sent_by
```

Comment:
Why setting sentBy prop and immediately deleting it?

Reply:
From the backend we receive sent_by I pass it to sentBy (camelCase) and always deleting the initial part of the string that I think does not contribute anything of value "message_sent_by_" I only keep the final part "bot", "agent" or "user" at the end of the string.